danom.gemspec

```diff
@@ -11,17 +11,9 @@ Gem::Specification.new do |spec|
 
   spec.summary       = %q{Monads implemented the Ruby way}
   # spec.description   = %q{TODO: Write a longer description or delete this line.}
-  spec.homepage      = "TODO: Put your gem's website or public repo URL here."
+  spec.homepage      = "https://github.com/D3MNetworks/danom"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.

[Read more about it here.](http://www.rubydoc.info/github/bbatsov/rubocop/Rubocop/Cop/Style/StringLiterals)